plugins/trace_api_plugin/test/include/eosio/trace_api/test_common.hpp

```diff
@@ -92,11 +92,27 @@ namespace eosio::trace_api {
          lhs.data == rhs.data;
    }
 
+   bool operator==(const action_trace_v1& lhs, const action_trace_v1& rhs) {
+      return
+         lhs.global_sequence == rhs.global_sequence &&
+         lhs.receiver == rhs.receiver &&
+         lhs.account == rhs.account &&
+         lhs.action == rhs.action &&
+         lhs.authorization == rhs.authorization &&
+         lhs.data == rhs.data &&
+         lhs.return_value == rhs.return_value;
+   }
+
    bool operator==(const transaction_trace_v0& lhs,  const transaction_trace_v0& rhs) {
       return
          lhs.id == rhs.id &&
          lhs.actions == rhs.actions;
    }
+   bool operator==(const transaction_trace_v2& lhs,  const transaction_trace_v2& rhs) {
```

Comment:
I don't see the equality operators for transaction_trace_v3 or block_trace_v3

Reply:
equality operators added for them.